src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
-const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+const pushNotification = (x, y, title, description, type) => {
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification', type);
+
+  notification.style.position = 'absolute';
+  notification.style.left = `${x}px`;
+  notification.style.top = `${y}px`;
```

Comment:
Using absolute positioning for notifications might lead to overlap if multiple notifications are displayed at the same time. Consider using a different positioning strategy, such as relative positioning or stacking notifications with a margin.